.github/workflows/nodejs.yml

```diff
@@ -9,60 +9,15 @@ on:
     branches:
       - '**'
 
-env:
-  CI: true
-
 jobs:
-  lint:
-    name: Lint on Node.js ${{ matrix.node }} and ${{ matrix.os }}
-
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        node: [12]
-        os: [ubuntu-latest]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node }}
-
-      - name: Install latest npm
-        run: npm install --global npm@latest
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Lint
-        run: npm run lint
-
   test:
-    name: Test on Node.js ${{ matrix.node }}
-
     runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        node: [10, 12]
-
     steps:
       - uses: actions/checkout@v2
-
-      - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node }}
-
-      - name: Install latest npm
-        run: npm install --global npm@latest
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Test
-        run: npm test
+          node-version: '10'
+      - run: npm install --global npm@latest
```

Comment:
I think the names make the [output](https://github.com/stylelint/stylelint-demo/runs/2055130527?check_suite_focus=true) easier to consume as they hide away the actual commands in favour of human-friendly sentences.

Let's add (for the job):
- name: Lint and test

And for the steps:
- name: Install latest npm
- name: Install dependencies
- name: Lint
- name: Test

Reply:
Make sense. Fixed via a46e239. 👍

Reply:
Changed `Setup` to `Set up` for consistency:

<img width="202" alt="image" src="https://user-images.githubusercontent.com/473530/110325890-44d32680-805b-11eb-81c1-d52f02f56e1b.png">

https://github.com/stylelint/stylelint-demo/pull/245/checks?check_run_id=2056618741